code/modules/hydroponics/grown/misc.dm

```diff
@@ -157,6 +157,53 @@
 	reagents.chem_temp = 1000 //Sets off the black powder
 	reagents.handle_reactions()
 
+//Murmuring Tree
+/obj/item/seeds/murmuring_wood
+	name = "pack of murmuring tree seeds"
+	desc = "This seed grows into a murmuring tree."
+	icon_state = "seed-murmuring"
+	species = "murmuring"
+	plantname = "Murmuring Tree"
+	//doesn't use product, see harvest()
+	genes = list(/datum/plant_gene/trait/settled_traits)
+	lifespan = 20
+	endurance = 20
+	maturation = 40
+	production = 10
+	yield = 1
+	potency = 60
+	growthstages = 3
+	rarity = 60 // Just stick to buying it from cargo, OK?
```

Comment:
So will this only be acquirable from cargo, or will it be an expensive item in seed vending machines?

Reply:
Probably both, the first one is expensive in the machine and every after is cargo only